CHANGELOG_iOS.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.77.98](https://github.com/brave/brave-browser/releases/tag/v1.77.98)
+
+ - Implemented an exception list for Night Mode. ([#44213](https://github.com/brave/brave-browser/issues/44213))
+ - Improved the performance of searching using URL bar for users with many bookmarks, tabs and history items. ([#43703](https://github.com/brave/brave-browser/issues/43703))
+ - Improved search results filtering and auto-complete when using the URL bar. ([#44842](https://github.com/brave/brave-browser/issues/44842))
+ - Improved cosmetic filtering performance. ([#42530](https://github.com/brave/brave-browser/issues/42530))
+ - Updated default search engine to Yahoo! JAPAN for new installations in Japan region. ([#43311](https://github.com/brave/brave-browser/issues/43311))
+ - Updated Brave Translate to hide "contribution" element in reader mode. ([#44835](https://github.com/brave/brave-browser/issues/44835))
+ - Updated Brave Translate onboarding. ([#44650](https://github.com/brave/brave-browser/issues/44650))
+ - Updated the “Enable Global Privacy Control” settings toggle placement to be at the bottom of the advanced settings page. ([#43690](https://github.com/brave/brave-browser/issues/43690))
+ - Fixed crash when clearing a large amount of history. ([#44981](https://github.com/brave/brave-browser/issues/44981))
+ - Fixed normal tabs being removed when disabling the “Keep Private Tabs” setting toggle. ([#45350](https://github.com/brave/brave-browser/issues/45350))
+ - Fixed an issue where tabs are not blocking pop-ups after switching from another tab. ([#44419](https://github.com/brave/brave-browser/issues/44419))
+ - Fixed "Default Model View" page not loading under Brave Leo hamburger settings. ([#44072](https://github.com/brave/brave-browser/issues/44072))
+ - Fixed audio continuing to play for users that closed a tab which showed a JavaScript alert. ([#43699](https://github.com/brave/brave-browser/issues/43699))
+ - Fixed “Shred Site Data” not working while using reader mode. ([#43663](https://github.com/brave/brave-browser/issues/43663))
+ - Fixed several issues relating to blob URL spoofing as reported on HackerOne by b4dc4t and frozzipies. ([#43654](https://github.com/brave/brave-browser/issues/43654))
```

Comment:
Apologies, missed this one but we need to add `[Security]` which is done for H1 reports. Lets also move it to the top of the `Fixed` list as well.

```
- [Security] Fixed several issues relating to blob URL spoofing as reported on HackerOne by b4dc4t and frozzipies. ([#43654](https://github.com/brave/brave-browser/issues/43654))
```